lib/drivers/medtronic600/medtronic600Driver.js

```diff
@@ -1859,8 +1833,84 @@ module.exports = (config) => {
       data.minimedConnection = false;
       data.pumpHighSpeedMode = false;
 
-      progress(100);
-      cb(null, data);
+      const settings = {
+        units: {
+          bg: 'mg/dL', // Even though the pump can be in mmol/L, we read the mg/dL settings values
+          carb: 'grams',
+        },
+      };
+
+      // Wrap in async, because this function is not async, and we're dealing with other
+      // async functions
+      (async () => {
+        // The enter/exit/enter paradigm is what CareLink does, according to packet captures.
```

Comment:
Ha, I was wondering how they do it! I always start with exiting remote command mode before entering it again, as it seems the meter can get stuck in remote command mode if a previous communication has failed.